salt/metalk8s/orchestrate/certs/renew.sls

```diff
@@ -0,0 +1,18 @@
+{%- set target_pillar = salt.saltutil.runner(
```

Comment:
Why exactly does this need to be an orchestration? :thinking: Is this required for a reactor to work?

Reply:
Not exactly required, but in the good practices, they say:

> Matching and rendering Reactor SLS files is done sequentially in a single process. For that reason, reactor SLS files should contain few individual reactions (one, if at all possible). Also, keep in mind that reactions are fired asynchronously (with the exception of caller) and do not support requisites.

source: https://docs.saltstack.com/en/latest/topics/reactor/#best-practices-for-writing-reactor-sls-files

I assumed an orchestrate would be better.